.gitlab/kernel_matrix_testing/security_agent.yml

```diff
@@ -179,6 +181,8 @@ kernel_matrix_testing_run_security_agent_tests_arm64:
           - "fedora_38"
           - "debian_11"
           - "debian_12"
+          - "rocky_8.9"
```

Comment:
It's possible this will fail, because arm64 kernels before 5.8 are a bit spotty. If it does fail, feel free to remove it from this list